packages/frontend/src/components/dialogs/Transports/styles.module.scss

```diff
@@ -0,0 +1,81 @@
+.transportList {
+  flex-grow: 1;
+  overflow-y: auto;
+}
+
+.container {
+  margin: 0 10px 0 30px;
```

Comment:
This is not gonna work as intended together with "add transport" button in RTL mode.

<img width="533" height="420" alt="Image" src="https://github.com/user-attachments/assets/e2ceaafc-7a90-41f3-aa2b-f7b45af28fbc" />